Add docs on creating a new release pipeline for new release branches (#1104)
cloudfoundry-incubator/kubecf: doc/release/release-guidelines.md

)
```

## Concourse

If the pipeline is not there already, define, commit, push, and deploy a new
Concourse pipeline for the new release branch.

> Copy the kubecf.yaml config the new release branch to the list of branches:

Author: ```suggestion
Copy the kubecf.yaml config into a new config for the new release branch:
```
